modules/wayland/default.nix

```diff
@@ -62,10 +62,17 @@
   #  [[ "$(tty)" == /dev/tty1 ]] && ${nixpkgs.lib.getBin weston}/bin/weston
   #'';
 
-  system.userActivationScripts.setupDesktop = ''
-    mkdir -p ~/.config/labwc
-    cp --symbolic-link --update ${./menu.xml} ~/.config/labwc/menu.xml
-    cp --symbolic-link --update ${./autostart} ~/.config/labwc/autostart
-    echo '${if withNaturalKeyboard then "XKB_DEFAULT_LAYOUT=de" else ""}' > ~/.config/labwc/environment
-  '';
+  system.userActivationScripts.setupDesktop =
+    let
+      envFile = "~/.config/labwc/environment";
+    in
+    ''
+      mkdir -p ~/.config/labwc
+      cp --symbolic-link --update ${./menu.xml} ~/.config/labwc/menu.xml
+      cp --symbolic-link --update ${./autostart} ~/.config/labwc/autostart
+      cat ${./xkbMacKeyboardConfig} > ${envFile}
```

Comment:
What are the benefits of using `cat ${./xkbMacKeyboardConfig} > ${envFile}` as opposed t `cp ${./xkbMacKeyboardConfig} ${envFile}`?

Reply:
None. I was just so much in the cat mindset. Miau!